sentry-spring/src/main/java/io/sentry/spring/SentryHubRegistrar.java

```diff
@@ -56,6 +57,17 @@ private void registerSentryHubBean(BeanDefinitionRegistry registry) {
     registry.registerBeanDefinition("sentryHub", builder.getBeanDefinition());
   }
 
+  private void registerSentryExceptionResolver(
+      BeanDefinitionRegistry registry, AnnotationAttributes annotationAttributes) {
+    final BeanDefinitionBuilder builder =
+        BeanDefinitionBuilder.genericBeanDefinition(SentryExceptionResolver.class);
+    builder.addConstructorArgReference("sentryHub");
+    Integer order = annotationAttributes.getNumber("exceptionResolverOrder");
```

Comment:
```suggestion
    int order = annotationAttributes.getNumber("exceptionResolverOrder");
```

if theres a default value, it cant be null, right